.kokoro/tests/run_tests.sh

```diff
@@ -39,6 +39,7 @@ SECRETS_PASSWORD=$(cat "${KOKORO_GFILE_DIR}/secrets-password.txt")
 source ./testing/test-env.sh
 export GOOGLE_APPLICATION_CREDENTIALS=$(pwd)/testing/service-account.json
 export GOOGLE_CLIENT_SECRETS=$(pwd)/testing/client-secrets.json
+source "${KOKORO_GFILE_DIR}/automl_secrets.txt"
```

Comment:
@kurtisvg, I'm getting a key error for the environment variables. Is there something I missed on setting this one up?

Error: `KeyError: 'AUTOML_PROJECT_ID'`

Reply:
I think it's fixed. Ping me again if still an issue.